CONTRIBUTING.md

```diff
@@ -0,0 +1,20 @@
+Issues
+------
+
+If you've found a bug or have a great idea for a new feature that will help all users of impress.js, let us know by [adding your suggestion]
```

Comment:
Note "for a new feature that will help **all users** of impress.js". We don't want to create features that are too specific and only satisfies a small subset of impress.js users.